.github/workflows/multichain-e2e-template.yml

```diff
@@ -86,6 +85,7 @@ jobs:
           port-forward: true
           version: 0.2.20
           timeout: 30m
+          load-docker-images: ghcr.io/agoric/agoric-sdk:dev
```

Comment:
The commit using `load-docker-images` seems to be introduced before the new `starship-action` supporting it.